test/unit/models/forms/FormConstraintsTest.scala

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.forms
+
+import java.util.Calendar
+
+import play.api.data.validation.{Invalid, Valid}
+import uk.gov.hmrc.play.test.UnitSpec
+
+class FormConstraintsTest extends UnitSpec {
+
+  private val currentYear: Int = Calendar.getInstance().get(Calendar.YEAR)
+  //TODO read from appconfig
```

Comment:
Is this TODO still accurate?

Reply:
this will be fix as part of unit test refactor ticket